cirq/google/__init__.py

```diff
@@ -12,31 +12,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from cirq.google.engine import (
+    engine_from_environment,
+    Engine,
+    JobConfig,
+)
+
+from cirq.google.line import (
+    AnnealSequenceSearchStrategy,
+    GreedySequenceSearchStrategy,
+    line_on_device,
+    LinePlacementStrategy,
+)
+
+from cirq.google.sim import (
```

Comment:
nit: should we sort these by module name, or is there some other organization scheme at work here?